docker/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
Now that this is a "real" entrypoint, we should probably have a comment in there explaining that this entrypoint is only used for CI/local envs.

Reply:
in the compose file or the entrypoint script?

Reply:
The entrypoint script itself

Reply:
Done.